OpenRA.Game/Network/OrderManager.cs

```diff
@@ -132,9 +132,13 @@ public void TickImmediate()
 			Connection.Receive(
 				(clientId, packet) =>
 				{
+					// A mod switch or other event has pulled the ground from beneath us
+					if (disposed)
```

Comment:
This stops the shellmap from ticking when joining a server... which I don't understand at all. I would understand not processing orders, but this also stops the camera movement that is done directly in tick?!

Reply:
Suggested quick fix so we can move on with more important things:
```
// HACK: The shellmap relies on ticking a disposed OM
if (disposed && World.Type != WorldType.Shellmap)
	return;
```